setup_undercloud.yml

```diff
@@ -45,6 +45,46 @@
         os_install: "RHEL {{ osp_rhel_mapping[osp_release] }}"
       when: force_reprovision == true
 
+    - name: clone badfish
+      git:
+        repo: "https://github.com/redhat-performance/badfish.git"
+        dest: "{{ ansible_user_dir }}/badfish"
+
+    - name: Create badfish virtualenv
+      command: python3 -m venv {{ badfish_venv }} creates={{ badfish_venv }}
+
+    - name: Install badfish requirements
+      pip:
+        requirements: "{{ ansible_user_dir }}/badfish/requirements.txt"
+        virtualenv_command: /usr/bin/python3 -m venv
+        virtualenv: "{{ badfish_venv }}"
+
+    - name: list oc_instackenv_content
+      shell: |
+        echo "{{ oc_instackenv_content.nodes[item | int].pm_addr }}"
+      with_sequence: 0-{{ (oc_instackenv_content.nodes|length - 1) }}
```

Comment:
Why do you want to set the director boot order for undercloud node?

Reply:
oc_instackenv_content does not include undercloud node